cli/tests/e2e/snapshots/test_ci/test_github_ci_bad_base_sha/results.txt

```diff
@@ -38,15 +38,11 @@ Trying to fetch branch foo from origin as base branch
 Trying to fetch branch bar as commit from origin as head branch tip commit
 Using f7ee3123ed098941662685dd8b726fceee48f78d as the merge-base of f7ee3123ed098941662685dd8b726fceee48f78d and 17b3114fb3d41437abc2a3e6499db84ad921e6f0
 Fetching configuration from semgrep.dev
+Getting scan configurations
 Starting scan
-Using https://semgrep.dev<MASKED>
-Adding ignore patterns configured on semgrep.dev as `--exclude` options: ()
+Authenticated as org_name
 semgrep version <MASKED>
-Fetching rules from https://semgrep.dev<MASKED>
-trying to download from https://semgrep.dev<MASKED>
-finished downloading from https://semgrep.dev<MASKED>
-loaded 1 configs in<MASKED>
-running 5 rules from 1 config remote-url_0
+running 5 rules from 1 config rules-yaml-string
```

Comment:
Yes, because it used to download from a rules.yaml endpoint and now it doesn't need to do that because it gets the rules from the same endpoint where it gets deployment id and name.

Reply:
Can we have it say semgrep.dev or something? rules-yaml-string will probably confuse users

Reply:
Yep! Technically this could be re-used for something besides app down the road, but changed for now